mantidproject/autoreduce
# Added delete functionality for future variables

Fixes #174 

To Test:
* Add a new set of future variables (by run number)
* Confirm that there is a delete button for these variables but not for the current ones
* Delete these variables
* Confirm that the page is refreshed and the variables are no longer there
* Add a new set of future variables (by RB number) (this only works with RB numbers that are scheduled on the instrument, e.g. 1510491 on LET)
*Confirm that deleting these also works
